src/main/java/utils/GcpClient.java

```diff
@@ -0,0 +1,45 @@
+package utils;
```

Comment:
We should start using proper packages starting with `io.cdap.test.e2e`

Reply:
package name of main folder renamed to io.cdap.e2e.utils and io.cdap.e2e.pages